zapier/triggers/list_workspaces.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  operation: {
+    perform: {
+      headers: { Accept: 'application/json' },
+      removeMissingValuesFrom: { body: false, params: false },
+      url: '{{process.env.BASE_URL}}/api/zapier/workspaces',
+    },
+    sample: { id: 1, name: 'My Workspace' },
+    outputFields: [
```

Comment:
Can we only show the workspace name? Let's keep IDs "internal"
![Workspace](https://github.com/user-attachments/assets/3b6c9619-c78f-4cb2-b88a-7e1ec11e8156)